### ISSUE TITLE
Basic functionality not working, does not delete target files.

### ISSUE THREAD

**Issue by insidewhy** [TarVK/ts-cleaner#1]
Seems like the basic functionality is broken:

```bash
% ls src/fats.ts
ls: cannot access 'src/fats.ts': No such file or directory
% npx ts-cleaner -d lib           
% ls lib/fats.js
lib/fats.js
```

Even though I deleted `src/fats.ts` I can still find `lib/fats.js`.

**Comment by TarVK** [comment]
Hi ohjames,

My apologies for the late reply, and thanks for the issue.
I verified that ts-cleaner doesn't work in the situation you provided.
I use the existence of `*.d.ts` files in the lib folder to determine whether it was generated from typescript. Of course not all users generate `*.d.ts` files however, so it doesn't always work, which is something I didn't consider.
In your situation, if you also create a `src/fats.d.ts` file, it does work.

In any case, I agree it should work like you described by default, and have some option to force how it currently works (for if people have js files copied to this dir directly as well).

I will try to add this soon!

**Comment by TarVK** [comment]
The described changes have been made in this commit:
https://github.com/TarVK/ts-cleaner/commit/0b24ace368ccd8f82d075e06317862b7a992eafc
And released this as version 1.0.3 on npm.

I hope this solves your issues.
